x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.tsx

```diff
@@ -72,50 +98,32 @@ export class Embeddable extends AbstractEmbeddable<LensEmbeddableInput, LensEmbe
   } = {};
 
   constructor(
-    timefilter: TimefilterContract,
-    expressionRenderer: ReactExpressionRendererType,
-    getTrigger: UiActionsStart['getTrigger'] | undefined,
-    {
-      expression,
-      savedVis,
-      editPath,
-      editUrl,
-      editable,
-      indexPatterns,
-    }: LensEmbeddableConfiguration,
+    private deps: LensEmbeddableDeps,
     initialInput: LensEmbeddableInput,
     parent?: IContainer
   ) {
     super(
       initialInput,
       {
-        defaultTitle: savedVis.title,
-        savedObjectId: savedVis.id,
-        editable,
-        // passing edit url and index patterns to the output of this embeddable for
-        // the container to pick them up and use them to configure filter bar and
-        // config dropdown correctly.
         editApp: 'lens',
-        editPath,
-        editUrl,
-        indexPatterns,
```

Comment:
Hopefully you'll find a way to fix the bug with indexPatterns!

Reply:
Fixed thanks to @Dosant in https://github.com/elastic/kibana/pull/76356